pyproject.toml

```diff
@@ -0,0 +1,37 @@
+[build-system]
+requires = [
+    "setuptools>=45",
+    "setuptools_scm>=7", 
+    "tomli>=1.0.0; python_version < '3.11'",
+]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools_scm]
+write_to = "dsps/_version.py"
```

Comment:
you need to specify the template per the example here: https://github.com/regro/conda-forge-metadata/blob/main/pyproject.toml

I'd just copy everything from that example and then change the lines you need to match.